fetch-ledger-develop-image/Dockerfile

```
@@ -56,6 +56,11 @@ RUN apt-get install -y libboost-all-dev=1.65.1.0ubuntu1 && \
     apt-get install -y libpng-dev && \
     apt-get install -y zlib1g-dev
 
+# Required for integration tests
+RUN apt-get install -y python3-pip
```

Unless it's needed for some reason it's best to reduce the steps as far as possible since each step is built as a new layer.

For example:
```suggestion
RUN apt-get install -y python3-pip && \
         pip3 install PyYAML && \
         pip3 install -U fetchai-ledger-api
```

There are a lot of RUN commands in this file.